src/main/resources/assets/language/hu.properties

```diff
@@ -0,0 +1,116 @@
+# ViaProxy language file
+#
+# Language contribution guidelines:
+# - Please use the same file content structure as in the en_US language file.
+# - Use the en_US language file as a reference/template for your language file.
+# - Keep the translations understandable and precise.
+# - Do not use or rely on machine translation (Google Translate, DeepL, etc.) exclusively.
+# - Do not copy an english translation into your language file. The en_US translation will be used as a fallback automatically.
+# - Include this header (untranslated) in your language file.
+
+# The display name of the language
+language.name=Hungarian
+# The name of the author of the language file (Your name)
+language.author=TheAdamNew
+
+generic.cancel=Mégsem
+generic.could_not_open_url=Nem lehet meg nyitni a linket: %s
+
+tab.general.name=Általános
+tab.general.server_address.label=Szerver IP Cím:
+tab.general.server_address.tooltip=Támogatott Cím:\n- address\n- address:port\n- ClassiCube Direct URL cím
+tab.general.server_version.label=Szerver Verzió:
+tab.general.minecraft_account.label=Minecraft Fiók:
+tab.general.minecraft_account.option_no_account=Ne használj semmilyen fiókot
+tab.general.minecraft_account.option_select_account=Kiválasztott fiók használata
+tab.general.minecraft_account.option_openauthmod=OpenAuthMod Használata
+tab.general.betacraft_auth.label=BetaCraft Auth(Classic)
+tab.general.betacraft_auth.tooltip=BetaCraft bekapcsolása. Fel enged Classic minecraft szerverekre.
+tab.general.state.loading=ViaProxy Betöltése...
+tab.general.state.start=Indít
```

Comment:
deepl.com translates this to "Indicate" which is wrong, it should mean Start

Reply:
i only used deepl for some things that i didn't know